### PR TITLE
Fix failing cite filter and black tests

### PR DESCRIPTION
**Pull request by agitter**
Update authors in Papernot et al. reference

### CODE REVIEW

**Comment by agitter** [comment]
I ran black 19.10b0 locally and the output was"68 files left unchanged."  However, black 20.8b1 (the version that runs in the CI tests), found these changes in 8 files.

File: conftest.py [hunk 1]
```diff
@@ -32,18 +32,18 @@ def pytest_addoption(parser):
...
-       - By default the cache persists for 12 hours. Default can be overrriden
-         with --requests-cache-hours option.
-       - Cache does not work for invocations of manubot commands as subprocess.
+    Notes:
+    - By default the cache persists for 12 hours. Default can be overrriden
+      with --requests-cache-hours option.
+    - Cache does not work for invocations of manubot commands as subprocess.
 
-       Cache file location: ./.cache/requests-cache.sqlite
+    Cache file location: ./.cache/requests-cache.sqlite
```
Comment (dhimmel) [inline conftest.py:46]: From https://black.readthedocs.io/en/stable/change_log.html

> Black now reindents docstrings when reindenting code around it (https://github.com/psf/black/pull/1053)

**Review by dhimmel — APPROVED** [review]
not sure which black change is resulting in long lines being split.

@agitter do you want to either merge or rebase merge such that these commits are kept separate?

thanks for fixing that CI failure.

**Comment by agitter** [comment]
@dhimmel yes, I'll do a regular merge to keep these commits separate

**Pull request merged**